firefox-ios/Client/Frontend/Autofill/Address/AddressListViewModel.swift

```diff
@@ -0,0 +1,64 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import SwiftUI
+import Common
+import Shared
+import Storage
+
+// TODO: PHASE-2 FXIOS-7653
+// AddressListViewModelDelegate: A protocol to notify delegates about address updates.
+// protocol AddressListViewModelDelegate: AnyObject {
+//     func didUpdateAddresses(_ addresses: [Address])
+// }
+
+extension Address {
+    var fullName: String {
+        return "\(givenName) \(familyName)"
+    }
+
```

Comment:
This seems to be a very North American centered way of formatting names and addresses. Will this be a feature that is only available in North America? If not we should probably use the users preferences to format both using `PersonNameComponentsFormatter` and `CNPostalAddressFormatter` for example.

Reply:
Yes phase 1 is north america

Reply:
If it is already planned to roll it out to more places in the future we should prepare for this - otherwise it might be forgotten. It is good practice to use formatters in any case.

Reply:
https://mozilla-hub.atlassian.net/browse/FXIOS-8337 ticket done for that